postpic/datahandling.py

```diff
@@ -447,6 +462,10 @@ def __getitem__(self, key):
         a slice containing floats or integers or a float or an integer
         """
         sl = self._normalize_slice(key)
+
+        if not isinstance(sl, slice):
+            return self.grid[sl]
```

Comment:
This does not really make sense but also does not really matter. Unclear what one should expect from this operation, anyhow.

Reply:
why did you add it then?

Reply:
On second thought, it makes perfect sense. For an `a = array([5,4,3])` it is totally  fine to return an `array([5,4])` on `a[:2]` and a plain `5` on `a[0]`. This just leads to `Axis` behaving the same way for integer indices. So in the integer case it's perfectly fine.
For real indices it will round to the neares grid point, so if there is `b = Axis(grid=[1,2,3])` then `b[1.8]` will return `2`. This might be a bit strange, but in the end who is to say that it's wrong.